### PR TITLE
Onblur

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/jepn128kbvddauywfcrtfgfue4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9095)

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: screen/wallets/import.js [hunk 1]
```diff
@@ -52,9 +52,10 @@ const WalletsImport = () => {
     },
   });
 
-  const _setImportText = value => {
-    const valueWithSingleWhitespace = value.replace(/^\s+|\s+$|\s+(?=\s)/g, '');
+  const onBlur = () => {
+    const valueWithSingleWhitespace = importText.replace(/^\s+|\s+$|(\r\n|\r|\n)|\s+(?=\s)/g, '');
```
Comment (limpbrains) [inline screen/wallets/import.js:56]: @marcosrdz 

I guess now SLIP39 wallet import is broken. It requires a few seeds divided by '\n'

What is even this code for? It is better to have all text preparations in main import function and be tested.

**Review by limpbrains — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: screen/wallets/import.js (hunk 1 above)
Comment (marcosrdz) [reply screen/wallets/import.js]: Reverted. This is so if the user uses the iOS OCR feature to "scan" their seed, any junk included gets filtered out.

Comment (marcosrdz) [reply screen/wallets/import.js]: Not sure about moving it to the import function since this is UI adjustment rather than "business logic".

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/r4vv7vatek13xxr9ygh1z9dnur?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9102)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/rhue8w162ezb5nn07gtd35xeb8?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9103)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/eqg9rmnkw5nr2g3151tq3qz8n8?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/9105)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Review by Overtorment — APPROVED** [review]

**Pull request merged**